src/row.js

```diff
@@ -62,6 +62,22 @@ export class Row {
     this.enemyFaction = row[47];
   }
 
+  getComposition = brackets => {
+    // We could have added all 5 classes and then filter out falsy values, but it would not be resilient to "ghost player" bugs
+    let arr = [this.enemyPlayerClass1, this.enemyPlayerClass2];
+    if (brackets.includes('5s'))
+      arr.push(
+        this.enemyPlayerClass3,
+        this.enemyPlayerClass4,
+        this.enemyPlayerClass5
+      );
+    else if (brackets.includes('3s')) arr.push(this.enemyPlayerClass3);
```

Comment:
It would be more logical and readable to me to add enemyPlayerClass3 if 3s or 5s and then push enemyPlayerClass4 and enemyPlayerClass5 if 5s

Reply:
yes but then you need an additional condition: if (5s but not 3s) you need to put enemyPlayerClass3 too, that's why it seemed better to go in reverse order